common/solar_system_initializers/hfe_initializer_features.txt

```diff
@@ -35,7 +35,7 @@ hfe_init_features_sanctuary_0 = {
 					min = 0
 					max = 1
 				}
-				class = random_non_colonizable
```

Comment:
Its intended to have fix amount of colonizable planet in system

Reply:
The Sanctuary should have more random variations for colonies. They shouldn't be equal.

Reply:
Would you like me to change it to depend on the presence or absence of Hardcore Mode?

Reply:
I'll set it to reply on Hardcore Mode being present or absent. You're right that they should not be guaranteed to spawn in Hardcore Mode.

Reply:
Okay, done